docs/src/operations/resource-planning.md

```diff
@@ -9,58 +9,57 @@ While we cannot tell you exactly what you need - beyond _it depends_ - we can ex
+- `segments` - the data of the log split into segments. The log is only appended - its data can be truncated when it becomes part of a new snapshot.
 - `state` - the active state. Deployed workflows, active workflow instances, etc. Completed workflow instances or jobs are removed.
-- `snapshot` - the data of a state at a certain point in time
+- `snapshot` - a state at a certain point in time
 
 If you want a recipe to explode your disk space usage, here are a few ways to do it:
 
-- Create a high number of snapshots with a long period between them.
 - Load an exporter, such as the Debug Exporter, that does not advance its record position.
 
 ## Event Log
 
-The event log for each partition is segmented. By default, the segment size is 512MB. This can be changed in the zeebe.cfg.toml file with the setting [logSegmentSize](https://github.com/zeebe-io/zeebe/blob/0.20.0/dist/src/main/config/zeebe.cfg.toml#L148). 
+The event log for each partition is segmented. By default, the segment size is 512MB. This can be changed in the zeebe.cfg.toml file with the setting [logSegmentSize](https://github.com/zeebe-io/zeebe/blob/0.20.0/dist/src/main/config/zeebe.cfg.toml#L148).
 
 An event log segment can be deleted once all the events it contains have been processed by exporters, replicated to other brokers, and processed. Three things can cause the event log to not be truncated:
 
-- A cluster loses its quorum, in which case events are queued but not processed. 
+- A cluster loses its quorum, in which case events are queued but not processed.
 - An exporter does not advance its read position in the event log.
-- The max number of snapshots has not been written.
+- There is no snapshot of it yet.
 
-An event log segment is not deleted until all the events in it have been exported by all configured exporters. This means that exporters that rely on side-effects, perform intensive computation, or experience back pressure from external storage will cause disk usage to grow, as they delay the deletion of event log segments. 
+An event log segment is not deleted until all the events in it have been exported by all configured exporters. This means that exporters that rely on side-effects, perform intensive computation, or experience back pressure from external storage will cause disk usage to grow, as they delay the deletion of event log segments.
 
 Exporting is only performed on the partition leader, but the followers of the partition do not delete segments in their replica of the partition until the leader marks all events in it as unneeded by exporters.
 
-No event log segments are deleted until the maximum number of snapshots has been reached. When the maximum number of snapshots has been reached, the event log is truncated up to the oldest snapshot.
+No event log segments are deleted until a snapshot has been taken that includes that log segment. When a snapshot has been taken, the event log is truncated up to that point.
```

Comment:
> No event log segments are deleted until a snapshot has been taken that includes that log segment

sounds a bit weird

Reply:
The original line reads to me as 'don't worry about your data being deleted too early'.  I've tried to capture that again. Perhaps it should just start like: 
```suggestion
We make sure that event log segments are not deleted too early. No event log segments are deleted until a snapshot has been taken that includes that segment. When a snapshot has been taken, the event log is only deleted up to that point.
```

Reply:
> No event log segments are deleted until a snapshot has been taken that includes that segment

Maybe:

No event log segment is deleted until a snapshot has been taken that includes that segment.

Reply:
That's better. So like:
```suggestion
We make sure that event log segments are not deleted too early. No event log segment is deleted until a snapshot has been taken that includes that segment. When a snapshot has been taken, the event log is only deleted up to that point.
```

Reply:
I think this is better thanks